msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/crosscloud/TestCase1592468.java

```diff
@@ -70,7 +73,7 @@ public TestCase1592468(final String name, final String guestHomeAzureEnvironment
     @Parameterized.Parameters(name = "{0}")
     public static Collection guestHomeAzureEnvironment() {
         return Arrays.asList(new Object[][]{
-                {"AZURE_US_GOV", LabConstants.GuestHomeAzureEnvironment.AZURE_US_GOV, /*homeCloud*/"https://login.microsoftonline.us", /*crossCloud*/"https://login.microsoftonline.com"},
+                {"AZURE_US_GOV", GuestHomeAzureEnvironment.AZURE_US_GOVERNMENT, /*homeCloud*/"https://login.microsoftonline.us", /*crossCloud*/"https://login.microsoftonline.com"},
```

Comment:
should this be GuestHomeAzureEnvironment.AZURE_US_GOVERNMENT.name() instead of just GuestHomeAzureEnvironment.AZURE_US_GOVERNMENT

Reply:
I think it should be toString(), actually. But yes. Stuff like this should show up once I re run automation. It looks like the checks did not catch this, which is odd.